src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
 
 export default class Fall_GM extends React.Component {
 	render() {
-		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT 
-	
-		const renderHumanReadableTime = ({ days, hours, minutes, seconds, completed }) => {
-			if (completed) return <h1 id="time-to-gm-headline-now">Fall General Meeting 2020 is happening right now!</h1>; //EVAN TODO: add the rest of the program and links to zoom / slack here
-	
-			let dayString ="Day";
-			let hourString = "Hour";
-			let minuteString = "Minute";
-			let secondString = "Second";
-	
-			if (days !== 1) dayString += 's';
-			if (hours !== 1) hourString += 's';
-			if (minutes !== 1) minuteString += 's';
-			if (seconds !== 1) secondString += 's';
+		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT
```

Comment:
To test, one can replace this line with
```
const testDate = new Date();
testDate.setSeconds(testDate.getSeconds() + 10);
const FALL_GM_2020_START_TIME = testDate;
```

Reply:
Quickly - can we move this `const` out of the `render` declaration? That way, we won't be redeclaring the GM time every re-render. May be helpful to move it to a constants file, or at least as a top-level constant.